apps/src/blockTooltips/DropletTooltipManager.js

```diff
@@ -65,22 +65,18 @@ DropletTooltipManager.prototype.installTooltipsOnVisibleToolboxBlocks = function
     }
 
     var funcName = $(blockHoverDiv).attr('title');
-    $(blockHoverDiv).tooltipster($.extend({}, DEFAULT_TOOLTIP_CONFIG, {
-      content: self.getDropletTooltip(funcName).getTooltipHTML()
-    }));
+
+    var hoverDivWidth = $(blockHoverDiv).width();
+    var hoverDivLeftToToolboxRight = $(".droplet-palette-canvas").width()
+      - parseInt(blockHoverDiv.style.left, 10);
```

Comment:
Not sure if our style guide says anything about this, but I prefer the operator at the end of the previous line.